<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>model_main</class>
 <widget class="QMainWindow" name="model_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>model_main</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-radius: 2px;
    border: 1px solid #3873d9;
  </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_any_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ª√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pi">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>pi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comma">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_degree_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>a²</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modul">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>|a|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_any_degree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
   </string>
    </property>
    <property name="text">
     <string>nª</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(170, 170, 127);
font: 9pt &quot;Wide Latin&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_answer">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delenie">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: white;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_umnozshenie">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ostatok">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 21pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_all">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>clear all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
